libkbfs/backpressure_disk_limiter.go

```diff
@@ -470,6 +479,49 @@ func (bdl *backpressureDiskLimiter) onBlocksDelete(
 	bdl.fileTracker.onBlocksDelete(blockFiles)
 }
 
+func (bdl *backpressureDiskLimiter) onDiskBlockCacheDelete(
+	ctx context.Context, blockBytes int64) {
+	if blockBytes == 0 {
+		return
+	}
+	bdl.lock.Lock()
+	defer bdl.lock.Unlock()
+	bdl.byteTracker.onBlocksDelete(blockBytes)
+}
+
+func (bdl *backpressureDiskLimiter) beforeDiskBlockCachePut(
+	ctx context.Context, blockBytes, diskBlockCacheBytes int64) (
```

Comment:
Okay, let's discuss this on Monday, then...also, https://github.com/keybase/kbfs/pull/837 might help things somewhat, since it imposes a limit that is more stringent than anything we have now, e.g. if the quota is 10 GB, then the journal will most likely not go very much past that, and we can let the disk cache eat up everything else.